globals.go

```diff
@@ -291,6 +291,9 @@ const (
 	// Valid types of performing git clone
 	CloneMethodLegacySSH = "legacy-ssh"
 	CloneMethodOAuth     = "oauth"
+
+	// PublicRepoName is the repo name for which a project ref is considered public.
```

Comment:
Small suggestion to define what "public" means in this context. I would just say something like it has an exemption from auth requirements for REST routes?